lib/SectionsManager.jsm

```diff
@@ -435,7 +435,7 @@ class SectionsFeed {
         if (action.data) {
           const matched = action.data.name.match(/^(feeds.section.(\S+)).options$/i);
           if (matched) {
-            SectionsManager.addBuiltInSection(matched[1], action.data.value);
+            await SectionsManager.addBuiltInSection(matched[1], action.data.value);
```

Comment:
It might be safer to remove the await in `storage.get` inside this function rather than this since it's not strictly necessary, although I think this might be fine too